<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameBoardPage</class>
 <widget class="QMainWindow" name="gameBoardPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="backgroundBoardLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>961</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/gallery.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="user1Label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="user2Label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff;
text-align: center;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="subText">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Source Code Pro</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(209, 209, 209);</string>
    </property>
    <property name="text">
     <string>chose your tactical board</string>
    </property>
   </widget>
   <widget class="QPushButton" name="board">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/board1.jpg</normaloff>:/images/board1.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>420</width>
      <height>250</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="nextBoardButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>200</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/next.png</normaloff>:/images/next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>38</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="prevBoardButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/images/prev.png</normaloff>:/images/prev.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>38</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
